core/src/main/java/io/fabric8/maven/core/service/ApplyService.java

```diff
@@ -972,6 +964,20 @@ public boolean applyNamespace(Namespace entity) {
         return false;
     }
 
+    /**
+     * Creates and return a project in openshift
+     * @param project
+     * @return
+     */
+    public boolean applyProject(Project project) {
+/*        getOpenShiftClient().projects().
```

Comment:
If it is not necessary just remove it

Reply:
aah I'm sorry I forgot to omit this part